src/decorators.js

```diff
@@ -55,10 +55,15 @@ export function customElement(name: string): any {
 /**
 * Decorator: Indicates that the decorated class is a custom attribute.
 * @param name The name of the custom attribute.
-* @param defaultBindingMode The default binding mode to use when the attribute is bound wtih .bind.
+* @param defaultBindingMode The default binding mode to use when the attribute is bound with .bind.
+* @param aliases The array of aliases to associate to the custom attribute.
 */
-export function customAttribute(name: string, defaultBindingMode?: number): any {
+export function customAttribute(name: string, defaultBindingMode?: number, aliases?: string[]): any {
   return function(target) {
+    if (aliases) {
+      target.aliases = aliases;
```

Comment:
The only reason why I compared it to inject, is that it is simple to access and modify. If we get something as simple as a static property I'm totally happy. Going the decorator way isn't the solution though, as that is only for the creator of the Custom Attribute, but not the consumer, whereas the most interesting use case for Aliases is actually the Consumer being able to kinda *rename* the attribute to fit it's own dev pattern.